### PR TITLE
[IMP] point_of_sale: daily sales report layout improved

### PR DESCRIPTION
**Pull request by ksku-odoo**
Before this commit:
===================
image

After this commit:
===================
image

task: 4035954

### CODE REVIEW

File: addons/point_of_sale/views/point_of_sale_report.xml [hunk 1]
```diff
@@ -10,11 +10,28 @@
...
+        <field name="page_width">0</field>
+        <field name="orientation">Portrait</field>
+        <field name="margin_top">10</field>
+        <field name="margin_bottom">32</field>
+        <field name="margin_left">4</field>
+        <field name="margin_right">4</field>
+        <field name="header_line" eval="False" />
+        <field name="header_spacing">8</field>
+        <field name="dpi">90</field>
+    </record>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/point_of_sale_report.xml:27]: @caburj 
Is it okay to create a new paperformat to achieve this?

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 2]
```diff
@@ -35,32 +71,26 @@
...
                                     <t t-if="currency['position']">
                                         <span t-out="category['total']" t-options="{'widget': 'float', 'precision': currency['precision']}">10000.00</span><span t-out='currency["symbol"]'>$</span>
                                     </t>
                                     <t t-else="" >
                                         <span t-out='currency["symbol"]'>$</span><span t-out="category['total']" t-options="{'widget': 'float', 'precision': currency['precision']}">10000.00</span>
                                     </t>
                                 </td>
-                                <tr t-foreach="category['products']" t-as="line">
-                                    <td/>
+                                <tr style="border-bottom: 0.1px solid #ccc;" t-foreach="category['products']" t-as="line">
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:90]:
```suggestion
                                <tr t-foreach="category['products']" t-as="line" style="border-bottom: 0.1px solid #ccc;">
```

**Review by paso-odoo — CHANGES_REQUESTED** [review]

**Comment by paso-odoo** [comment]
@ksku-odoo 
- Removed newly created paper format
- Header with minimal code
I allowed myself to push the commit in your PR for the above points.
Please have a look at it!

Thanks

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 3]
```diff
@@ -149,39 +166,30 @@
                         </tbody>
                     </table>
                     <div class="oe_structure"></div>
-                </t>
+                </div>
                 <t t-if="refund_products" id="refunds">
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:170]:
```suggestion
                </div>
                <div t-if="refund_products" id="refunds" style="page-break-inside: avoid;">
```

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 4]
```diff
@@ -149,39 +166,30 @@
...
-                                    <td/>
+                                <tr t-foreach="category['products']" t-as="line" style="border-bottom: 0.1px solid #ccc;">
                                     <t t-set="internal_reference" t-value="line['code'] and '[%s] ' % line['code'] or ''" />
-                                    <td><span t-out="internal_reference">7897</span><span t-out="line['product_name']">Laptop</span></td>
-                                    <td>
-                                        <span t-out="internal_reference">DEMO_REF</span>
-                                        <span t-out="line['product_name']">DEMO_PRODUCT_NAME</span>
-                                    </td>
+                                    <td style="padding-left: 40px;"><span t-out="internal_reference">7897</span><span t-out="line['product_name']">Laptop</span></td>
+                                    <td/>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:192]:
```suggestion
                                    <td><span t-out="internal_reference">7897</span><span t-out="line['product_name']">Laptop</span></td>
                                    <td/>
```
avoid style and use bootstrap

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 5]
```diff
@@ -222,16 +231,11 @@
...
 
                 <t t-if="refund_taxes" id="taxe_refunds">
-                    <h3>Taxes on refunds</h3>
-                    <table  class="table table-sm">
-                        <thead><tr>
-                            <th class="text-start">Name</th>
-                            <th class="text-end">Tax Amount</th>
-                            <th class="text-end">Base Amount</th>
-                        </tr></thead>
+                    <h6 class="custom-header fw-bold w-50 ms-auto">Taxes on refunds</h6>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:234]: `refund_taxes` should be in div and avoid it to break the page

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 6]
```diff
@@ -299,7 +299,7 @@
                     </t>
                 </t>
                 <div id="discounts" style="break-inside: avoid; page-break-inside: avoid;">
-                    <h3>Discounts</h3>
+                    <h6  class="custom-header fw-bold">Discounts</h6>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:302]:
```suggestion
                    <h6 class="fw-bold">Discounts</h6>
```

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 7]
```diff
@@ -275,15 +279,11 @@
                 <div class="oe_structure"></div>
                 <t t-if="payments" id="payments">
                     <t t-if="payments" id="other_payments">
-                        <h3>Payments</h3>
-                        <table class="table table-sm">
-                            <thead><tr>
-                                <th class="text-start">Name</th>
-                                <th class="text-end">Total</th>
-                            </tr></thead>
+                        <h6 class="custom-header fw-bold w-50 ms-auto">Payments</h6>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:282]: `payments` should also be in the div with avoiding the page-break

File: addons/point_of_sale/views/pos_session_sales_details.xml [hunk 8]
```diff
@@ -321,140 +321,136 @@
...
-            <t t-if="invoiceTotal" id="invoices">
-                <br/>
-                <h3>Invoices</h3>
-                <table  class="table table-sm">
-                    <thead><tr>
-                        <th class="text-start">Name</th>
-                        <th class="text-start">Order reference</th>
-                        <th class="text-end">Total</th>
-                    </tr></thead>
-                    <tbody>
```
Comment (paso-odoo) [inline addons/point_of_sale/views/pos_session_sales_details.xml:335]: Is there any reason for removing this code?

**Review by paso-odoo — CHANGES_REQUESTED** [review]

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**